### ISSUE TITLE
questions about loss？

### ISSUE THREAD

**Issue by Dr-Zhou** [jvanvugt/pytorch-domain-adaptation#3]

**Comment by Dr-Zhou** [comment]
why the loss don't include critic loss?The paper final loss =classifier loss + wasserstain loss +gradient penalty loss.

**Comment by weiyikang** [comment]
I have the same confusion. The model is initiated by the pre-trained model on the source domain, which is different from the original paper.
